sdk/containerregistry/container-registry/swagger/containerregistry.json

```diff
@@ -45,18 +45,12 @@
       "description": "Blob Operations"
     }
   ],
-  "schemes": [
-    "https"
-  ],
-  "produces": [
-    "application/json"
-  ],
+  "schemes": ["https"],
+  "produces": ["application/json"],
   "paths": {
     "/v2/": {
       "get": {
-        "tags": [
-          "Registry"
-        ],
+        "tags": ["Registry"],
         "description": "Tells whether this Docker Registry instance supports Docker Registry HTTP API v2",
         "operationId": "ContainerRegistry_CheckDockerV2Support",
```

Comment:
Would it be possible to make this internal so this method doesn't get generated on the client?  We don't currently have a customer scenario for this operation.  If not, we can hide it in the customization layer - just let us know!

Reply:
I added `"x-accessibility": "internal",` but it doesn't seem to have any effect.  I can deleted it from swagger directly or via a transformation I guess.  For TS we choose what to expose so we don't need to hide things.  @seankane-msft and @pallavit do you choose to expose or hide in Python/Java?

Reply:
Thanks for looking into that!  We'll remove this in the customization layer for .NET.

Reply:
@jeremymeng Python will hide this in the customization layer.